scipy/optimize/_minimize.py

```diff
@@ -205,6 +208,14 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
     and either the Hessian or a function that computes the product of
     the Hessian with a given vector. Suitable for large-scale problems.
 
+    Method :ref:`trust-krylov <optimize.minimize-trustkrylov>` uses
+    the Newton GLTR trust-region algorithm [13]_ for unconstrained
```

Comment:
Definitely, I have added the references. It is also better as one has not to find the bits and pieces in various places of the Trust Region Textbook but has a shorter publication reference.